Smart-Contract-Verifier-Server/src/main/kotlin/com/savannasolutions/SmartContractVerifierServer/negotiation/services/NegotiationService.kt

```diff
@@ -240,7 +264,8 @@ class NegotiationService constructor(val agreementsRepository: AgreementsReposit
 
         val agreement = agreementsRepository.getById(setPaymentConditionRequest.AgreementID)
 
-        if(setPaymentConditionRequest.PreposedUser != agreement.partyA.publicWalletID && setPaymentConditionRequest.PreposedUser != agreement.partyB.publicWalletID)
```

Comment:
I just noticed the spelling error in Proposed. This will affect the API, right?

Reply:
Nop. As I have to specify the JSON property in the request object the spelling mistake has already been corrected there